File: linear-search/LinearSearch.java
public class LinearSearch {
    public static int linearSearch(int[] arr, int target) {
        for (int i = 0; i < arr.length; i++) {
            if (arr[i] == target) return i;
        }
        return -1;
    }
    
    public static void main(String[] args) {
        int[] arr = {5, 2, 8, 1, 9, 3};
        System.out.println(linearSearch(arr, 8));
        System.out.println(linearSearch(arr, 7));
    }
}

File: linear-search/linear-search.kt
fun linearSearch(arr: IntArray, target: Int): Int {
    for (i in arr.indices) {
        if (arr[i] == target) return i
    }
    return -1
}

fun main() {
    val arr = intArrayOf(5, 2, 8, 1, 9, 3)
    println(linearSearch(arr, 8))
    println(linearSearch(arr, 7))
}
